src/WurzelBot.py

```diff
@@ -34,6 +36,10 @@ def __init__(self):
         self.storage = Storage(self.__HTTPConn)
         self.garten = []
         self.wassergarten = None
+        self.bienenfarm = None
+        self.bonsaifarm = None
+        self.bonsai = Bonsai(self.__HTTPConn)
+        self.honig = Honig(self.__HTTPConn)
```

Comment:
Gibt es hier Duplikate?
self.bonsai <---> self.bonsaifarm
self.honig <---> self.bienenfarm

Reply:
kann sein , das habe ich selber nicht ganz verstanden...

du legst ja oben zum Beispiel 'self.wassergarten = None' fest
und dann 
```
if self.spieler.isAquaGardenAvailable() is True:
self.wassergarten = AquaGarden(self.__HTTPConn)
```
Deshalb dachte muss ich es auch nochmal extra zuordnen

Reply:
Ich habe die Objektattribute deklariert und mit None initialisiert. Wenn der Bot startet werden die dann korrekt zugewiesen, sofern es den entsprechenden Teil im Spiel gibt (z. B. den Wassergarten). Man könnte mit der Abfrage auf None die Abfrage auf isAvailable wahrscheinlich sparen. Darüber kann man durchaus diskutieren.

Das doppelte Anlegen von Variablen hingegen ist nicht notwendig und eher hinderlich, weil es verwirrt. Hier empfehle ich dringend eine Vereinheitlichung. In Objekten werden nämlich auch Daten abgelegt. Wenn die doppelten Objekte dann unterschiedliche Daten haben ist das einfach nur pures Chaos.

Ich würde im init das Objektattribut mit None belegen und später zuweisen. Wie ich es vorher auch gemacht habe. Dann ist es immerhin in einem einheitlichen Stil implementiert und kann später auch besser aufgeräumt werden.